test/parallel_api/numeric/numeric.ops/inclusive_scan_by_segment.pass.cpp

```diff
@@ -27,8 +27,82 @@ using namespace oneapi::dpl::execution;
 #endif
 using namespace TestUtils;
 
+//#define DUMP_CHECK_RESULTS
+
+namespace
+{
+#if TEST_DPCPP_BACKEND_PRESENT
+    // accessors
+    template <typename T, int Dim, sycl::access::mode AccMode, sycl::access::target AccTarget,
+              sycl::access::placeholder Placeholder>
+    T*
+    get_host_iterator(sycl::accessor<T, Dim, AccMode, AccTarget, Placeholder>& acc)
+    {
+        return &acc[0];
```

Comment:
@MikeDvorskiy  FYI : in our version of compiler and SYCL library begin() method still absent.

/home/ksa_build_dpcpp/.vs/github-oneapi/cf6d1844-0499-4cd7-a7c1-c4d2b9e5ac0d/src/test/parallel_api/numeric/numeric.ops/inclusive_scan_by_segment.pass.cpp:39:5: note: candidate template ignored: substitution failure [with T = unsigned long, Dim = 1, AccMode = sycl::access::mode::read_write, AccTarget = sycl::access::target::host_buffer, Placeholder = sycl::access::placeholder::false_t]: no member named 'begin' in 'sycl::accessor<unsigned long, 1, sycl::access::mode::read_write, sycl::access::target::host_buffer, sycl::access::placeholder::false_t, sycl::ext::oneapi::accessor_property_list<>>'
    get_host_iterator(sycl::accessor<T, Dim, AccMode, AccTarget, Placeholder>& acc)